gulp/conf.js

```diff
@@ -39,3 +29,47 @@ exports.errorHandler = function(title) {
     this.emit('end');
   };
 };
+
+/**
+ * Most straightforward way to load dependencies in order with gulp
+ * Need to move to webpack
```

Comment:
Assuming temporary until we migrate to webpack.

Reply:
correct